Branch-SDK/src/main/java/io/branch/referral/ServerRequestRegisterInstall.java

```diff
@@ -61,6 +61,9 @@ public void onPreExecute() {
             if(installReferrerServerTS > 0){
                 getPost().put(Defines.Jsonkey.InstallBeginServerTimeStamp.getKey(), installReferrerServerTS);
             }
+
+            JSONObject configurations = Branch.getInstance().getConfigurationController().serializeConfiguration();
+            getPost().put("operational_metrics", configurations);
```

Comment:
There's a class `Defines.java` which contains an enum class `Jsonkey`, add `operational_metrics` there. These are the keys to the json fields we send in requests, and is an easy place for others to reference.